pkgs/development/compilers/gcc/9/default.nix

```diff
@@ -88,7 +88,7 @@ stdenv.mkDerivation ({
 
   inherit patches;
 
-  outputs = [ "out" "man" "info" ] ++ stdenv.lib.optional (!langJit) "lib";
+  outputs = [ "out" "man" "info" ] ++ stdenv.lib.optional (!langJit && targetPlatform == hostPlatform) "lib";
```

Comment:
we definitely want the lib output in cross so that our cross closures don't accidentally pull in the compiler

Reply:
I’m not questioning that. This is just what breaks it right now.